src/test/cephtool-test-mon.sh

```diff
@@ -16,6 +16,8 @@
 # GNU Library Public License for more details.
 #
 CEPH_CLI_TEST_DUP_COMMAND=1 \
+# uses ports 7202 7203 and 7204 because
```

Comment:
If this line included "CEPH_PORT" and listed the 3 ports numbers, I would have seen it.  I was using "git grep CEPH_PORT" to find out what ports were being used.